### ISSUE TITLE
[Discussions]Do we need to add a document for this template?

### ISSUE THREAD

**Issue by sgalcheung** [florian-lefebvre/astro-integration-template#151]
Usually, people must add a doc to introduce the integration, so I think a doc is also necessary. and I'm writing an integration, I found this template doesn't have a doc folder.

**Comment by florian-lefebvre** [comment]
Yeah it's true I need to improve the documentation. I'll be on holiday soon so I think I'll take some time to update how this is structured and documented

**Comment by sgalcheung** [comment]
Enjoy your holiday!
